src/stratis_cli/_actions/_logical.py

```diff
@@ -66,10 +66,10 @@ def list_volumes(namespace):
         pool_object_path = \
            GetObjectPath.get_pool(proxy, spec={'Name': namespace.pool})
 
-        for object_path, _ in get_managed_objects(proxy).filesystems(
+        for _, info in get_managed_objects(proxy).filesystems(
                    props={'Pool': pool_object_path}
            ):
-            print(object_path)
+            print(info['org.storage.stratis1.filesystem']['Name'])
```

Comment:
```stratisd_client_dbus``` package defines two classes, ```GMOFilesystem``` and ```GMOPool```. The idea is that they get at the table, like: 

```print(GMOFilesystem(info).Name())```

which is a bit more principled. The same should work below, with ```GMOPool(info).Name()```. These should continue to exist once the whole thing gets generalized, so they might as well be used.